src/Modules/Admin/Commands.Admin/help/Get-PowerBIActivityEvents.md

```diff
@@ -0,0 +1,107 @@
+---
+external help file: Microsoft.PowerBI.Commands.Admin.dll-Help.xml
```

Comment:
I have a work item created for adding documentation: https://powerbi.visualstudio.com/Power%20BI/_workitems/edit/201026
If this is something which is required for initial checkin of this PR, I will go ahead and update the docs as well.